crates/iota-indexer/src/types.rs

```diff
@@ -416,6 +416,27 @@ impl EventIndex {
     }
 }
 
+#[cfg(feature = "pg_integration")]
```

Comment:
I would suggest exposing this for all tests:

```suggestion
#[cfg(test)]
```

Reply:
`#[cfg(test)]` is only enabled on unit tests, tests marked as `#[test]` under the `src` directory.

Tests under the `tests` directory are treated as integration tests, and they’re treated as independent crates that have the library as a dependency. Items with `#[cfg(test)]` will not be compiled.

As an alternative, I could add this:
```rust
// compiles when either condition is true
#[cfg(any(test, feature = "pg_integration"))]
```

Reply:
fixed: https://github.com/iotaledger/iota/pull/7969/commits/5f601e42b9603eab8de7de10f29c082f5f622b75